apps/issuer/v1_api_urls.py

```diff
@@ -10,6 +10,15 @@
     url(r'^all-badges$', AllBadgeClassesList.as_view(), name='v1_api_issuer_all_badges_list'),
     url(r'^all-badges/find$', FindBadgeClassDetail.as_view(), name='v1_api_find_badgeclass_by_identifier'),
 
+    url(r'^all-superbadges$', AllSuperBadgeClassesList.as_view(), name='v1_api_issuer_all_superbadges_list'),
+    url(r'^all-collectionbadges$', AllCollectionBadgeClassesList.as_view(), name='v1_api_issuer_all_collectionbadges_list'),
+
+    url(r'^collectionbadges/(?P<slug>[-\w]+)$', CollectionBadgeDetail.as_view(),
+        name='v1_api_collectionbadges_detail'),
+
+    # url(r'^superbadges/(?P<slug>[-\w]+)$', SuperBadgeDetail.as_view(),
```

Comment:
Comment?

Reply:
This endpoint is not ready for use yet, the ```SuperBadgeDetail``` class is not implemented yet. I added a TODO comment to clarify this